src/patches/Briefing.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BattleTech;
+using BattleTech.Data;
+using BattleTech.Rendering;
+using BattleTech.Rendering.Mood;
+using BattleTech.UI;
+using Harmony;
+using HBS;
+using HBS.Collections;
+
+namespace EnvironmentalDesignMasks
+{
+    [HarmonyPatch(typeof(Briefing), "Init")]
+    public static class Briefing_Init
+    {
+        private static void Postfix(Briefing __instance, LoadingCamera ___loadCam)
+        {
+            try
+            {
+                var moodName = UnityGameInstance.BattleTechGame?.Combat?.ActiveContract?.mapMood;
+                if (moodName == null) return;
```

Comment:
```suggestion
                if (moodName == null) {
                    return;
                }
```

This is a preference thing - I do not like single line conditionals. Just matching the repo's existing style again.